README.md

```diff
@@ -98,26 +99,32 @@ The event name and what's in the `data` for each event handler:
   * `event` The raw event
   * `senderId` The ID of the sender
   * `session` [A Session object](#the-session-object) you can mutate
-  * `text` Message content, `event.message.text` for text events
+  * `payload` Message content, `event.message.text` for text events
```

Comment:
Just to be clear, I'm ok with breaking backwards compatibility too as long as we decide together. NAMINGTHINGS is hard.